browser/javascript/react-app/src/App.js

```diff
@@ -111,6 +111,10 @@ function App() {
           Group
         </button>
 
+        <button onClick={() => ampli.groupIdentify('test-group-2', 'a-group-value-2', { requiredBoolean: true })}>
```

Comment:
Actually, can we do groupIdentify on the same group we set with the user? Basically same group we call `setGroup`. The groupIdentify call won't automatically bind the group to the user, so I can't see the group properties set when check on the user. This would apply to other examples as well. Thank you.